widgetssdk/src/main/java/com/glia/widgets/permissions/domain/PermissionsForMediaUpgradeOfferUseCase.kt

```diff
@@ -0,0 +1,23 @@
+package com.glia.widgets.permissions.domain
+
+import android.Manifest
+import android.annotation.SuppressLint
+import android.os.Build
+import com.glia.androidsdk.comms.MediaDirection
+import com.glia.androidsdk.comms.MediaUpgradeOffer
+import com.glia.widgets.permissions.Permissions
+
+internal class PermissionsForMediaUpgradeOfferUseCase(
```

Comment:
May it be a good idea to name it `GetPermissionsForMediaUpgradeOfferUseCase` or `PreparePermissionsForMediaUpgradeOfferUseCase`?